src/test/custom_loader_test.cpp

```diff
@@ -211,24 +178,9 @@ const char* custom_loader_config_model_multiple = R"({
       ]
     })";
 
-class MockModel : public ovms::Model {
-public:
-    MockModel() :
-        Model("MOCK_NAME") {}
-    MOCK_METHOD(ovms::Status, addVersion, (const ovms::ModelConfig&), (override));
-};
-
-std::shared_ptr<MockModel> modelMock;
 }  // namespace
 
-class MockModelManager : public ovms::ModelManager {
-public:
-    std::shared_ptr<ovms::Model> modelFactory(const std::string& name) override {
-        return modelMock;
-    }
-};
-
-class TestCustomLoader : public ::testing::Test {
+class TestCustomLoader : public TestPredict {
 public:
     void SetUp() {

```

Comment:
You made this child class of TestPredict but you are not calling base class SetUp method which I think is not intended:

you most probably should call:
TestPredict::Setup().

cleaning up temporary destination most probably should be done in TearDown method?(at the end of test)

Reply:
The Setup in original calls TestPredict::Setup() sets up model instance and that is not used in customloader tests. these tests start from file. The Setup in derived class does the needed for customloader tests. So, we did not call base class setup.

The teardown actually deletes the files and does cleanup. We have put the other cleanup call in setup just incase preious test execution returned abnormally and some files remained.